### DISCUSSION TITLE
Kibana Issues

### DISCUSSION THREAD

**Discussion by palitech100** [Security-Onion-Solutions/securityonion#7529] [Unsupported Versions] [answered]
Hello,

I am having some issues with our standalone system. I had the following errors after running soup. Elastaler containers shows as missing and I have issues with Kibana. I would very much appreciate help. Thank you in advance!

   ID: wait_for_kratos
    Function: http.wait_for_successful_query
        Name: http://palindrome-so:4434/
      Result: True
     Comment: Status 404 was found.
                status: 404
                error: HTTP 404: Not Found
     Started: 15:37:27.715130
    Duration: 16.71 ms

          ID: wait_for_elasticsearch
    Function: cmd.run
        Name: so-elasticsearch-wait
      Result: False
     Comment: Command "so-elasticsearch-wait" run
     Started: 15:55:00.767773
    Duration: 371093.025 ms
     Changes:

     ID: so-elastalert
    Function: docker_container.running
      Result: False
     Comment: One or more requisite failed: elastalert.wait_for_elasticsearch
     Started: 16:01:11.906708
    Duration: 0.007 ms
     Changes:

Succeeded: 732 (changed=37)
Failed:      2
--------------
Total states run:     734
Total run time:  1486.798 s
Soup failed with error 1: Unhandled error
Unhandled error occured, please check /root/soup.log for details.

I am not able to pull up Kibana. I get 404 page not found. Diagnostic logs from Elasticsearch and Kibana are below.

/opt/so/log/elasticsearch
Caused by: org.elasticsearch.action.search.SearchPhaseExecutionException: Search rejected due to missing shards [[.kibana_7.17.1_001][0]]. Consider using `allow_partial_search_results` setting to bypass this error.
        at org.elasticsearch.action.search.AbstractSearchAsyncAction.run(AbstractSearchAsyncAction.java:227) ~[elasticsearch-7.17.1.jar:7.17.1]
        at org.elasticsearch.action.search.AbstractSearchAsyncAction.executePhase(AbstractSearchAsyncAction.java:454) [elasticsearch-7.17.1.jar:7.17.1]
        ... 189 more
[2022-03-11T20:28:40,299][WARN ][org.elasticsearch.gateway.PersistedClusterStateService] writing cluster state took [11105ms] which is above the warn threshold of [10s]; wrote global metadata [false] and metadata for [1] indices and skipped [999] unchanged indices
[2022-03-11T20:28:40,301][INFO ][org.elasticsearch.cluster.coordination.Coordinator.CoordinatorPublication] after [10.9s] publication of cluster state version [34627] is still waiting for {palindrome-so}{OzCEeb7eTWO7uqn4rRA4eA}{rvMEOsCQShuradcxO1CLIw}{palindrome-so}{192.168.1.22:9300}{cdfhimrstw}{box_type=hot, xpack.installed=true, transform.node=true} [SENT_PUBLISH_REQUEST]
[2022-03-11T20:30:29,691][WARN ][org.elasticsearch.gateway.PersistedClusterStateService] writing cluster state took [10406ms] which is above the warn threshold of [10s]; wrote global metadata [false] and metadata for [1] indices and skipped [999] unchanged indices
[2022-03-11T20:30:29,693][INFO ][org.elasticsearch.cluster.coordination.Coordinator.CoordinatorPublication] after [10.5s] publication of cluster state version [34642] is still waiting for {palindrome-so}{OzCEeb7eTWO7uqn4rRA4eA}{rvMEOsCQShuradcxO1CLIw}{palindrome-so}{192.168.1.22:9300}{cdfhimrstw}{box_type=hot, xpack.installed=true, transform.node=true} [SENT_PUBLISH_REQUEST]

/opt/so/log/kibana/
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-03-11T20:26:17.426+00:00","message":"[.kibana_task_manager] Action failed with 'search_phase_execution_exception: '. Retrying attempt 15 in 64 seconds.","log":{"level":"ERROR","logger":"savedobjects-service"},"process":{"pid":293}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-03-11T20:26:17.427+00:00","message":"[.kibana_task_manager] OUTDATED_DOCUMENTS_SEARCH_OPEN_PIT -> OUTDATED_DOCUMENTS_SEARCH_OPEN_PIT. took: 64056ms.","log":{"level":"INFO","logger":"savedobjects-service"},"process":{"pid":293}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-03-11T20:26:17.904+00:00","message":"[.kibana] Action failed with 'search_phase_execution_exception: '. Retrying attempt 15 in 64 seconds.","log":{"level":"ERROR","logger":"savedobjects-service"},"process":{"pid":293}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-03-11T20:26:17.905+00:00","message":"[.kibana] OUTDATED_DOCUMENTS_SEARCH_OPEN_PIT -> OUTDATED_DOCUMENTS_SEARCH_OPEN_PIT. took: 64054ms.","log":{"level":"INFO","logger":"savedobjects-service"},"process":{"pid":293}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-03-11T20:27:21.472+00:00","message":"[.kibana_task_manager] OUTDATED_DOCUMENTS_SEARCH_OPEN_PIT -> FATAL. took: 64046ms.","log":{"level":"INFO","logger":"savedobjects-service"},"process":{"pid":293}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-03-11T20:27:21.475+00:00","message":"Unable to complete saved object migrations for the [.kibana_task_manager] index: Unable to complete the OUTDATED_DOCUMENTS_SEARCH_OPEN_PIT step after 15 attempts, terminating.","error":{"message":"Unable to complete saved object migrations for the [.kibana_task_manager] index: Unable to complete the OUTDATED_DOCUMENTS_SEARCH_OPEN_PIT step after 15 attempts, terminating.","type":"Error","stack_trace":"Error: Unable to complete saved object migrations for the [.kibana_task_manager] index: Unable to complete the OUTDATED_DOCUMENTS_SEARCH_OPEN_PIT step after 15 attempts, terminating.\n    at migrationStateActionMachine (/usr/share/kibana/src/core/server/saved_objects/migrationsv2/migrations_state_action_machine.js:144:29)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Promise.all (index 1)\n    at SavedObjectsService.start (/usr/share/kibana/src/core/server/saved_objects/saved_objects_service.js:181:9)\n    at Server.start (/usr/share/kibana/src/core/server/server.js:330:31)\n    at Root.start (/usr/share/kibana/src/core/server/root/index.js:69:14)\n    at bootstrap (/usr/share/kibana/src/core/server/bootstrap.js:120:5)\n    at Command.<anonymous> (/usr/share/kibana/src/cli/serve/serve.js:229:5)"},"log":{"level":"FATAL","logger":"root"},"process":{"pid":293}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-03-11T20:27:21.496+00:00","message":"Stopping all plugins.","log":{"level":"INFO","logger":"plugins-system.standard"},"process":{"pid":293}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-03-11T20:27:21.498+00:00","message":"Monitoring stats collection is stopped","log":{"level":"INFO","logger":"plugins.monitoring.monitoring.kibana-monitoring"},"process":{"pid":293}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-03-11T20:27:21.947+00:00","message":"[.kibana] OUTDATED_DOCUMENTS_SEARCH_OPEN_PIT -> FATAL. took: 64043ms.","log":{"level":"INFO","logger":"savedobjects-service"},"process":{"pid":293}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-03-11T20:27:51.506+00:00","message":"\"eventLog\" plugin didn't stop in 30sec., move on to the next.","log":{"level":"WARN","logger":"plugins-system.standard"},"process":{"pid":293}}

**Comment by dougburks** [comment, marked as answer]
What version were you on before?

Are you able to share the full soup.log?

**Reply by palitech100** [reply]
Hi Doug,

Thank you for responding. The previous version was. 2.3.100.  I am not sure what went wrong, but I rebooted the system a few times, restarted some of the containers, and reloaded Kibana dashboards and the problem is fixed :)

Thank you again.

Martina
